### ISSUE TITLE
[feature] webhook on various call events

### ISSUE THREAD

**Issue by beppe9000** [Chooloo/call_manage#42]
I would like to get a notification on my windows pc when there is an incoming call.

The easiest way to implement that is to have the app make a web request (HTTPS POST) somewhere when a call is received. The user is then free to setup their end to handle it however they see fit.

Example of payload for an incoming call:
```js
{eventName:"INCOMING_CALL",contactName:"John Doe", phoneNumber:"0123456789"}
```

I suggest to add two settings to this app:
 - A text field to specify the target url
 - A checkbox group to select what events are to be notified externally (e.g. incoming call, missed call, call ended)

**Comment by jumper047** [comment]
It is seems like work for Tasker

**Comment by beppe9000** [comment]
yeah, but having it builtin web events api is way cleaner

**Comment by talusannni** [comment]
You should create a socket server and connect with the mobile application.
Your socket server listens to events from mobile pp and notifies your web application
This way you have 2-way communication and can control calls from your web application.

socket server------------------------->Mobile application
    /|\  |          <------------------------
     |    |
     |    |
     |   \|/
Web application

**Comment by beppe9000** [comment]
Yes, my idea was something like that, to mimic AirDroid or PushBullet or Microsoft Android call alert.

> and connect with the mobile application

is there an existing facility for this ?
